tests/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import {assert, expect} from "chai";
+import {Int, Uint} from "../lib/Interfaces";
+import BigNumber from "bignumber.js";
+
+/**
+ * Executes fn and assert that the error the same message as expected. If no error is thrown, assert will fail.
+ * @param fn - The function to test
+ * @param {Error} expected - Error with the expected message
+ */
+export const assertErrorMsg = (fn , expected: Error) => {
+    let err = null;
+    try {
+        fn();
+    } catch (e) {
+        err = e;
+    }
+    if (!err) {
+        expect.fail("No error thrown");
+    } else {
+        expect(err.message).to.equal(expected.message);
+    }
+};
```

Comment:
The `if/else` doesn't need to be after,`try/catch` will handle all of that logic for you. The following should work.
```

export const assertErrorMsg = (fn , expected: Error) => {
    let err = null;
    try {
        fn();
        // If fn() throws, below will not reach
        expect.fail("No error thrown");
    } catch (e) {
        expect(err.message).to.equal(expected.message);
    }
};
```

Reply:
Excellent point, will update